lib/metricdatatypes/data.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metricdatatypes
+
+/*
+Data from UMA Export Server
+*/
+
+// UMAExportQuery represents the enumeration of queries that can be sent to the UMA Export Server.
+type UMAExportQuery string
+
+const (
+	WebDXFeaturesQuery UMAExportQuery = "usecounter.webdxfeatures"
+)
+
+// BucketDataMetric contains the metric details for a particular bucket.
+type BucketDataMetric struct {
+	Rate      float64 `json:"rate,omitempty"`
+	Milestone string  `json:"milestone,omitempty"`
```

Comment:
Should milestones be ints?

Reply:
Good question. `milestone` is the name of the field returned from the uma export server. But `HistogramEnumValue` has this problem. BucketID should really be `Value`. That would represent more closely what is in the enums.xml file. I have modified that.

Additionally, I split the struts into two files. One representing data from the uma export server. The other representing the data from the enums.xml